src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php

```diff
@@ -119,6 +132,12 @@ public function getLabel() {
    */
   public function getTokenTypes() {
     $definition = $this->getPluginDefinition();
+    // For some reason, we didn't unify token keys with entity types...
+    foreach ($definition['types'] as $key => $type) {
```

Comment:
Yeah. I also have a related fix in one of the pending PR's for bringing in 7.x-1.x improvements/fixes.